api/v1/routes/blog.py

```diff
@@ -1,18 +1,21 @@
+from typing import List
+
+from fastapi import APIRouter, Depends, HTTPException, status
 from fastapi.encoders import jsonable_encoder
-from api.v1.schemas.blog import BlogUpdateResponseModel, BlogRequest
 from sqlalchemy.orm import Session
+
 from api.db.database import get_db
+from api.utils.dependencies import get_current_user, get_super_admin
+from api.v1.models.blog import Blog
 from api.v1.models.user import User
-from api.utils.dependencies import get_current_user
+from api.v1.schemas.blog import (BlogRequest, BlogResponse,
+                                 BlogUpdateResponseModel)
 from api.v1.services.blog import BlogService
-from fastapi import APIRouter, Depends, HTTPException, status
-from typing import List
-from api.v1.models.blog import Blog
-from api.v1.schemas.blog import BlogResponse
 
-blog = APIRouter(prefix="/blogs", tags=["Blog"])
```

Comment:
hint: this is why other's test are failing. don't just blame others on your fault. if they don't make that test then thier endpoints would have been fallen without notice🙃

Reply:
Okay. Thanks alot. I have fixed it.